health/health.d/timex.conf

```diff
@@ -0,0 +1,17 @@
+
+# It can take several minutes before ntpd selects a server to synchronize with;
+# try checking after 17 minutes (1024 seconds).
+
+    alarm: system_clock_sync_state
+       on: system.clock_sync_state
+       os: linux
+    class: System
+component: Clock
+     type: Error
+     calc: $state
+    units: synchronization state
+    every: 10s
+     warn: $netdata.uptime.uptime > 17 * 60 AND $this == 0
+    delay: down 5m
+     info: the system time is not synchronized to a reliable server
+       to: silent
```

Comment:
@yasharne do you think we shouldn't make it _silent_ by default?

Reply:
Making it `silent` makes sense e.g. in our scenario only some of the servers need to have their `clock` monitored